plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -242,6 +245,30 @@ class ServicesTable extends React.Component {
     );
   }
 
+  renderRegions(prop, service) {
+    const localRegion = CompositeState.getMasterNode().getRegionName();
+    let regions = service.getRegions();
+
+    regions = regions.map(
+      region => (region === localRegion ? region + " (Local)" : region)
+    );
+
+    if (regions.length === 0) {
+      regions.push("N/A");
```

Comment:
We are starting to have this `N/A` all over the code base, I would rather have a stateless component that renders that. I also record that at some point last year we wanted to replace `N/A` to `-` and I believe that we created a helper or a component that returns `-`

Reply:
There was also a recent case where the design team preferred having a blank cell. Should we talk to design about how we decide when to use `N/A` vs `-` vs nothing?

Reply:
@mperrotti good point. I remember that case too now which was for services version.

Reply:
Yeah, I like the Idea of having small `<Empty />` and `<NotAvailable />` components to keep things aligned. For the empty case we also switch between `null`, `<div />` and `<span />` 😕

However I'd say it is not in scope for this feature branch to create these components, we can clean up the code later - @weblancaster @mperrotti can one of you create Epics and facilitate them? 🙇